Potential jMimeMagic bug

### Expected behavior
After runtime setup the installation file for the software should be executed.
### Actual behavior
I get this in terminal output alongside the GraalVM error mentioned in #1988 
```
[INFO ] org.springframework.context.annotation.ConfigurationClassParser (l.458) - Properties location [file:${application.user.settings}] not resolvable: /home/jonasz/.Phoenicis/config.properties (No such file or directory)
[INFO ] org.phoenicis.repository.DefaultRepositoryManager (l.126) - Adding repositories: [org.phoenicis.repository.location.LocalRepositoryLocation@e2a4236[/home/jonasz/Downloads/scripts-master], org.phoenicis.repository.location.GitRepositoryLocation@418cc584[https://github.com/PhoenicisOrg/scripts,master]] at index 0
[INFO ] org.phoenicis.repository.types.MultipleRepository (l.46) - Fetching applications for: org.phoenicis.repository.types.MultipleRepository@574cbb82[repositories=[org.phoenicis.repository.types.LocalRepository@598b4f1b[repositorySource=file:/home/jonasz/Downloads/scripts-master/,repositoryDirectory=/home/jonasz/Downloads/scripts-master], git-repository https://github.com/PhoenicisOrg/scripts, local folder: /home/jonasz/.Phoenicis/cache/git-331995911, branch: master]]
[INFO ] org.phoenicis.repository.types.GitRepository (l.76) - Begin fetching process of git-repository https://github.com/PhoenicisOrg/scripts, local folder: /home/jonasz/.Phoenicis/cache/git-331995911, branch: master
[INFO ] org.phoenicis.repository.types.LocalRepository (l.77) - Reading directory : /home/jonasz/Downloads/scripts-master
[INFO ] org.phoenicis.repository.types.GitRepository (l.145) - Opening git-repository https://github.com/PhoenicisOrg/scripts, local folder: /home/jonasz/.Phoenicis/cache/git-331995911, branch: master
[INFO ] org.phoenicis.repository.types.GitRepository (l.149) - Pulling new commits from git-repository https://github.com/PhoenicisOrg/scripts, local folder: /home/jonasz/.Phoenicis/cache/git-331995911, branch: master
[INFO ] org.phoenicis.repository.types.LocalRepository (l.77) - Reading directory : /home/jonasz/.Phoenicis/cache/git-331995911
[WARN ] org.phoenicis.javafx.views.mainwindow.installations.dto.InstallationDTO (l.56) - Installation ID (The Sims 2_1560199086081) contains invalid characters, will remove them.
[INFO ] org.phoenicis.tools.archive.Tar (l.56) - Uncompressing /home/jonasz/.Phoenicis/resources/sims2_fixes.tar.xz to dir /home/jonasz/.Phoenicis/containers/wineprefix/The Sims 2/drive_c/users/jonaszMy Documents/EA Games/The Sims™ 2 Ultimate Collection/Downloads.
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '320 kBits'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '384 kBits'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '\b3'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '\b2'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Mono'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '- %d frames,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'width=%d pixels,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'height=%d pixels,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'depth=%d,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'ticks/frame=%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '- %d frames'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'width=%d pixels,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'height=%d pixels,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'depth=%d,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'ticks/frame=%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ''
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '\b.%d,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d instruments'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d bits'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'ASCII,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'continuation,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'extra field,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'original filename,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'comment,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'encrypted,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'last modified: %s,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%c)'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%c)'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%ld x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%ld,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%ld x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%ld,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d-bit'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'interlaced'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %.3s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher ': %s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'POSIX'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %c'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '\b.%c'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'length %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 've'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d entries'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d entries'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d x'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'v%d'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher '%s'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'version %c,'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect keyboard file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect document'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect dictionary'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect thesaurus'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect block'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect rectangular block'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect column block'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect printer data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect printer data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect driver resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect hyphenation code'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect hyphenation data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect macro resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect hyphenation lex'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect wordlist'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect equation resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect spell rules'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect dictionary rules'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect spell rules (Microlytics)'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect settings file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect 4.2 document'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect dialog file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect button bar'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Shell macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Shell definition'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Notebook macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Notebook help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Notebook keyboard file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Notebook definition'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Calculator help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Calendar help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Calendar data file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Editor macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Editor help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Editor keyboard file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Editor macro resource file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Macro editor macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Macro editor help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Macro editor keyboard file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect macro'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect keyboard file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect worksheet'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect printer definition'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect graphic definition'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect temporary printer'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'PlanPerfect macro resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'distribution list'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'out box'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'in box'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'users archived mailbox'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'archived message database'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'archived attachments'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Printer temporary file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Scheduler help file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Scheduler in file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Scheduler out file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise settings file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise directory services'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise settings file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Terminal resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Terminal resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Terminal resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GUI loadable text'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'graphics resource data'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'printer settings file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'port definition file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'print queue parameters'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'compressed file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Network service msg file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Network service msg file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'Async gateway login msg'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise message file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise admin domain database'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise admin host database'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise admin remote host database'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'GroupWise admin ADS deferment data file'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'IntelliTAG (SGML) compiled DTD'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect graphic image (1.0)'
[INFO ] net.sf.jmimemagic.MagicParser (l.429) - endElement(): not adding invalid matcher 'WordPerfect graphic image (2.0)'
[ERROR] net.sf.jmimemagic.MagicMatcher (l.731) - failed to load detector: 
                net.sf.jmimemagic.detectors.TextFileDetector
            
java.lang.ClassNotFoundException: 
                net.sf.jmimemagic.detectors.TextFileDetector
            
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:315)
	at net.sf.jmimemagic.MagicMatcher.getDetectorExtensions(MagicMatcher.java:726)
	at net.sf.jmimemagic.Magic.initialize(Magic.java:90)
	at net.sf.jmimemagic.Magic.getMagicMatch(Magic.java:175)
	at org.phoenicis.tools.files.FileAnalyser.getMatch(FileAnalyser.java:67)
	at org.phoenicis.tools.files.FileAnalyser.getMimetype(FileAnalyser.java:83)
	at org.phoenicis.tools.archive.Extractor.uncompress(Extractor.java:59)
	at org.phoenicis.tools.archive.Extractor.uncompress(Extractor.java:45)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostMethodDesc$SingleMethod$MHBase.invokeHandle(HostMethodDesc.java:269)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostMethodDesc$SingleMethod$MHBase.invoke(HostMethodDesc.java:261)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode$1.executeImpl(HostExecuteNode.java:776)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.GuestToHostRootNode.execute(GuestToHostRootNode.java:87)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callInlinedForced(OptimizedCallTarget.java:276)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget$OptimizedCallInlined.call(OptimizedCallTarget.java:880)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.GuestToHostRootNode.guestToHostCall(GuestToHostRootNode.java:113)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.doInvoke(HostExecuteNode.java:786)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.doOverloadedCached(HostExecuteNode.java:215)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNodeGen.executeAndSpecialize(HostExecuteNodeGen.java:226)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNodeGen.executeImpl(HostExecuteNodeGen.java:94)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.execute(HostExecuteNode.java:97)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObject.invokeMember(HostObject.java:371)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObjectGen$InteropLibraryExports$Cached.invokeMemberNode_AndSpecialize(HostObjectGen.java:1308)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObjectGen$InteropLibraryExports$Cached.invokeMember(HostObjectGen.java:1283)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.invokeMember(InteropLibraryGen.java:3104)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$ForeignInvokeNode.executeCall(JSFunctionCallNode.java:1388)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$InvokeNode.execute(JSFunctionCallNode.java:708)
	at com.oracle.truffle.js.nodes.JavaScriptNode.executeVoid(JavaScriptNode.java:212)
	at com.oracle.truffle.js.nodes.control.AbstractBlockNode.executeVoid(AbstractBlockNode.java:74)
	at com.oracle.truffle.js.nodes.control.BlockNode.execute(BlockNode.java:62)
	at com.oracle.truffle.js.nodes.function.FunctionBodyNode.execute(FunctionBodyNode.java:66)
	at com.oracle.truffle.js.nodes.function.FunctionRootNode.executeInRealm(FunctionRootNode.java:147)
	at com.oracle.truffle.js.runtime.JavaScriptRealmBoundaryRootNode.execute(JavaScriptRealmBoundaryRootNode.java:93)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:239)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:63)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$UnboundJSFunctionCacheNode.executeCall(JSFunctionCallNode.java:1236)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$InvokeNode.execute(JSFunctionCallNode.java:708)
	at com.oracle.truffle.js.nodes.JavaScriptNode.executeVoid(JavaScriptNode.java:212)
	at com.oracle.truffle.js.nodes.control.AbstractBlockNode.executeVoid(AbstractBlockNode.java:74)
	at com.oracle.truffle.js.nodes.control.BlockNode.execute(BlockNode.java:62)
	at com.oracle.truffle.js.nodes.function.FunctionBodyNode.execute(FunctionBodyNode.java:66)
	at com.oracle.truffle.js.nodes.function.FunctionRootNode.executeInRealm(FunctionRootNode.java:147)
	at com.oracle.truffle.js.runtime.JavaScriptRealmBoundaryRootNode.execute(JavaScriptRealmBoundaryRootNode.java:93)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:239)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:63)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$UnboundJSFunctionCacheNode.executeCall(JSFunctionCallNode.java:1236)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$InvokeNode.execute(JSFunctionCallNode.java:708)
	at com.oracle.truffle.js.nodes.JavaScriptNode.executeVoid(JavaScriptNode.java:212)
	at com.oracle.truffle.js.nodes.control.AbstractBlockNode.executeVoid(AbstractBlockNode.java:74)
	at com.oracle.truffle.js.nodes.control.BlockNode.execute(BlockNode.java:62)
	at com.oracle.truffle.js.nodes.function.FunctionBodyNode.execute(FunctionBodyNode.java:66)
	at com.oracle.truffle.js.nodes.function.FunctionRootNode.executeInRealm(FunctionRootNode.java:147)
	at com.oracle.truffle.js.runtime.JavaScriptRealmBoundaryRootNode.execute(JavaScriptRealmBoundaryRootNode.java:93)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:239)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:63)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$UnboundJSFunctionCacheNode.executeCall(JSFunctionCallNode.java:1236)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$InvokeNode.execute(JSFunctionCallNode.java:708)
	at com.oracle.truffle.js.nodes.unary.VoidNodeGen.execute(VoidNodeGen.java:25)
	at com.oracle.truffle.js.nodes.function.FunctionBodyNode.execute(FunctionBodyNode.java:66)
	at com.oracle.truffle.js.nodes.function.FunctionRootNode.executeInRealm(FunctionRootNode.java:147)
	at com.oracle.truffle.js.runtime.JavaScriptRealmBoundaryRootNode.execute(JavaScriptRealmBoundaryRootNode.java:93)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:239)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:63)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$UnboundJSFunctionCacheNode.executeCall(JSFunctionCallNode.java:1236)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.interop.JSInteropExecuteNode.doDefault(JSInteropExecuteNode.java:72)
	at com.oracle.truffle.js.nodes.interop.JSInteropExecuteNodeGen.executeAndSpecialize(JSInteropExecuteNodeGen.java:63)
	at com.oracle.truffle.js.nodes.interop.JSInteropExecuteNodeGen.execute(JSInteropExecuteNodeGen.java:46)
	at com.oracle.truffle.js.runtime.truffleinterop.InteropBoundFunction.execute(InteropBoundFunction.java:111)
	at com.oracle.truffle.js.runtime.truffleinterop.InteropBoundFunctionGen$InteropLibraryExports$Cached.executeNode_AndSpecialize(InteropBoundFunctionGen.java:246)
	at com.oracle.truffle.js.runtime.truffleinterop.InteropBoundFunctionGen$InteropLibraryExports$Cached.execute(InteropBoundFunctionGen.java:228)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.execute(InteropLibraryGen.java:2626)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotValue$InteropCodeCache$AbstractExecuteNode.executeShared(PolyglotValue.java:1345)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotValue$InteropCodeCache$ExecuteVoidNoArgsNode.executeImpl(PolyglotValue.java:1399)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostToGuestRootNode.execute(HostToGuestRootNode.java:94)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget$OptimizedCallProfiled.call(OptimizedCallTarget.java:872)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotValue$InteropValue.executeVoid(PolyglotValue.java:2003)
	at org.graalvm.sdk/org.graalvm.polyglot.Value.executeVoid(Value.java:358)
	at org.phoenicis.scripts.Installer$$JSJavaAdapter.run(Unknown Source)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostMethodDesc$SingleMethod$MHBase.invokeHandle(HostMethodDesc.java:269)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostMethodDesc$SingleMethod$MHBase.invoke(HostMethodDesc.java:261)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode$1.executeImpl(HostExecuteNode.java:776)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.GuestToHostRootNode.execute(GuestToHostRootNode.java:87)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callInlinedForced(OptimizedCallTarget.java:276)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget$OptimizedCallInlined.call(OptimizedCallTarget.java:880)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.GuestToHostRootNode.guestToHostCall(GuestToHostRootNode.java:113)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.doInvoke(HostExecuteNode.java:786)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.doFixed(HostExecuteNode.java:136)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNodeGen.executeAndSpecialize(HostExecuteNodeGen.java:136)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNodeGen.executeImpl(HostExecuteNodeGen.java:94)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostExecuteNode.execute(HostExecuteNode.java:97)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObject.invokeMember(HostObject.java:371)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObjectGen$InteropLibraryExports$Cached.invokeMemberNode_AndSpecialize(HostObjectGen.java:1308)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.HostObjectGen$InteropLibraryExports$Cached.invokeMember(HostObjectGen.java:1283)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.invokeMember(InteropLibraryGen.java:3104)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$ForeignInvokeNode.executeCall(JSFunctionCallNode.java:1388)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeAndSpecialize(JSFunctionCallNode.java:292)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode.executeCall(JSFunctionCallNode.java:238)
	at com.oracle.truffle.js.nodes.function.JSFunctionCallNode$InvokeNode.execute(JSFunctionCallNode.java:708)
	at com.oracle.truffle.js.nodes.access.JSWriteCurrentFrameSlotNodeGen.execute_generic4(JSWriteCurrentFrameSlotNodeGen.java:177)
	at com.oracle.truffle.js.nodes.access.JSWriteCurrentFrameSlotNodeGen.execute(JSWriteCurrentFrameSlotNodeGen.java:93)
	at com.oracle.truffle.js.nodes.access.JSWriteCurrentFrameSlotNodeGen.executeVoid(JSWriteCurrentFrameSlotNodeGen.java:343)
	at com.oracle.truffle.js.nodes.control.ExprBlockNode.execute(ExprBlockNode.java:68)
	at com.oracle.truffle.js.nodes.function.FunctionBodyNode.execute(FunctionBodyNode.java:66)
	at com.oracle.truffle.js.nodes.function.FunctionRootNode.executeInRealm(FunctionRootNode.java:147)
	at com.oracle.truffle.js.runtime.JavaScriptRealmBoundaryRootNode.execute(JavaScriptRealmBoundaryRootNode.java:93)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:239)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:63)
	at com.oracle.truffle.js.lang.JavaScriptLanguage$1.execute(JavaScriptLanguage.java:212)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:325)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:315)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:302)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:284)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callIndirect(OptimizedCallTarget.java:227)
	at jdk.internal.vm.compiler/org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:214)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotContextImpl.eval(PolyglotContextImpl.java:800)
	at org.graalvm.sdk/org.graalvm.polyglot.Context.eval(Context.java:341)
	at org.graalvm.sdk/org.graalvm.polyglot.Context.eval(Context.java:367)
	at org.phoenicis.scripts.engine.implementation.PolyglotScriptEngine.eval(PolyglotScriptEngine.java:63)
	at org.phoenicis.scripts.interpreter.PhoenicisScriptInterpreter.runScript(PhoenicisScriptInterpreter.java:36)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$runScript$0(BackgroundScriptInterpreter.java:37)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[INFO ] org.phoenicis.tools.archive.Tar (l.172) - Creating output file /home/jonasz/.Phoenicis/containers/wineprefix/The Sims 2/drive_c/users/jonaszMy Documents/EA Games/The Sims™ 2 Ultimate Collection/Downloads/simNopke-simShadowFix0.4.package (420).
[INFO ] org.phoenicis.tools.archive.Tar (l.172) - Creating output file /home/jonasz/.Phoenicis/containers/wineprefix/The Sims 2/drive_c/users/jonaszMy Documents/EA Games/The Sims™ 2 Ultimate Collection/Downloads/thumbnails_fix.package (420).
```
### Steps to reproduce
Select Origin script for The Sims 2 https://github.com/PhoenicisOrg/scripts/pull/927 and try to run it.
### System information
- software version (git revision) of phoenicis and scripts repositories
b246b86
script repo PhoenicisOrg/scripts@53a6846
- used operating system
Ubuntu 19.04 x64
- used Java version
    openjdk 11.0.3 2019-04-16
    OpenJDK Runtime Environment (build 11.0.3+7-Ubuntu-1ubuntu219.04.1)
    OpenJDK 64-Bit Server VM (build 11.0.3+7-Ubuntu-1ubuntu219.04.1, mixed mode, sharing)

--- a/phoenicis-javafx/pom.xml
+++ b/phoenicis-javafx/pom.xml
@@ -127,7 +127,7 @@
         <dependency>
             <groupId>com.github.PhoenicisOrg</groupId>
             <artifactId>javafx-collections</artifactId>
-            <version>1.2.1</version>
+            <version>1.2.3</version>
         </dependency>
     </dependencies>
     <build>

--- a/phoenicis-tools/pom.xml
+++ b/phoenicis-tools/pom.xml
@@ -61,7 +61,7 @@
         <dependency>
             <groupId>com.github.PhoenicisOrg</groupId>
             <artifactId>jmimemagic</artifactId>
-            <version>0.2.3</version>
+            <version>0.3.3</version>
         </dependency>
         <dependency>
             <groupId>com.github.PhoenicisOrg</groupId>

